docs/conf.py

```diff
@@ -54,7 +54,7 @@ def format_annotation(annotation, *args, **kwargs):
 pygments_style = "sphinx"
 html_theme = "sphinx_rtd_theme"
 html_theme_options = {
-    "style_nav_header_background": "#8e6ca6",
+    "style_nav_header_background": "#343131",
```

Comment:
This makes the left pane a consistent color from top to bottom.

Reply:
Sounds good to me. The purple didn't look good.